src/OrchardCore/OrchardCore/Localization/BclCalendars.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using NodaTime;
+
+namespace OrchardCore.Localization
+{
+    internal static class BclCalendars
+    {
+        public static IEnumerable<Calendar> MappedCalendars =>
+            new[] { Hebrew, Hijri, Gregorian, Julian, Persian, UmAlQura };
+
+        public static Calendar Hebrew => new HebrewCalendar();
```

Comment:
Can we reuse the instances (readonly fields) or do we need new instances for every call?